src/main/resources/ca/corefacility/bioinformatics/irida/database/changesets/21.05/all-changes.xml

```diff
@@ -6,4 +6,6 @@
          http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
     <include file="project-default-metadata-template.xml"
              relativeToChangelogFile="true"/>
+    <include file="user-account-project-subscription.xml"
```

Comment:
Can you move this changeset to be included in 22.05 instead of 21.05

Reply:
Please see d26411b2f550895fb32bee12bd82f6339e648e4b & 681cc8d3fb768bf618e71609f861895b486fd40b.